docs/appendices/release-notes/unreleased.rst

```diff
@@ -109,8 +109,9 @@ Changes
   its SQL equivalent ``START TRANSACTION`` statement optional to support
   compatibility with clients and tools using an older (< 8.0) PostgreSQL syntax.
 
-- Updated to Admin UI 1.23.0, which improves scrolling behavior on wide result
-  sets.
+- Updated to Admin UI 1.23.1, which improves scrolling behavior on wide result
+  sets, and fixes formatting of ``TIMESTAMP WITHOUT TIME ZONE`` values in query
```

Comment:
```suggestion
  sets, and fixes formatting of 
  :ref:`TIMESTAMP WITHOUT TIME ZONE <type-timestamp-without-tz>` values in query
```

Reply:
I've added your suggestion to the patch. Thanks.